homeassistant/components/auth/__init__.py

```diff
@@ -273,15 +273,15 @@ async def _async_handle_revoke_token(self, hass, data):
         token = data.get("token")
 
         if token is None:
-            return web.Response(status=HTTP_OK)
+            return web.Response()
 
         refresh_token = await hass.auth.async_get_refresh_token_by_token(token)
 
         if refresh_token is None:
-            return web.Response(status=HTTP_OK)
+            return web.Response()
 
         await hass.auth.async_remove_refresh_token(refresh_token)
-        return web.Response(status=HTTP_OK)
+        return web.Response()
```

Comment:
The authentication code needs to be readable and not rely on defaults as much as possible. These explicit status assigments are on purpose.

Reply:
Restored in flagged cases.